server/db/migration/migrations/sqls/20201013053954-drop-country-extra-columns-up.sql

```diff
@@ -0,0 +1,15 @@
+alter table country drop column list_name_en;
+
+alter table country drop column full_name_en;
+
+alter table country drop column list_name_es;
+
+alter table country drop column full_name_es;
+
+alter table country drop column list_name_fr;
+
+alter table country drop column full_name_fr;
+
+alter table country drop column list_name_ru;
+
+alter table country drop column full_name_ru;
```

Comment:
should we drop also region_iso column ?

Reply:
It is dropped in another migration 
server/db/migration/migrations/sqls/20201001104113-country-remove-region-region-iso-up.sql